lib/src/flutterflow_api_client.dart

```diff
@@ -106,7 +110,8 @@ Future<String?> exportCode({
       extractArchiveToCurrentDirectory(projectFolder, destinationPath);
     }
 
-    folderName = projectFolder.first.name;
+    var fileName = projectFolder.first.name;
+    folderName = fileName.substring(0, fileName.indexOf('/'));
```

Comment:
Not sure if it'll work on Windows - use `Platform.pathSeparator` instead of `/`.